key.ui/src/main/java/de/uka/ilkd/key/gui/keyshortcuts/KeyStrokeSettings.java

```diff
@@ -178,7 +209,20 @@ public void save() {
         LOGGER.info("Save keyboard shortcuts to: {}", SETTINGS_FILE.getAbsolutePath());
         SETTINGS_FILE.getParentFile().mkdirs();
         try (Writer writer = new FileWriter(SETTINGS_FILE, StandardCharsets.UTF_8)) {
-            properties.store(writer, "KeY's KeyStrokes");
+            Properties props = new Properties();
+            for (Map.Entry<String, Object> entry : properties.entrySet()) {
+                props.setProperty(entry.getKey(), entry.getValue().toString());
+            }
+            props.store(writer, "KeY's KeyStrokes");
+            writer.flush();
+        } catch (IOException ex) {
+            ex.printStackTrace();
+        }
```

Comment:
Why are these settings saved twice?

Reply:
Ah okay. I guess that's also why recent-files is not saved in both formats (it is not important enough).